socs/agents/hi6200/drivers.py

```diff
@@ -12,14 +12,36 @@ class Hi6200Interface():
 
     def __init__(self, ip_address, tcp_port, verbose=False, **kwargs):
         """
-            Connects to the Hi6200 weight sensor using a TCP Modbus Client with pyModbusTCP.
-            This works ~similarly to a socket connection using a IP and port.
+            Connects to the Hi6200 weight sensor using a TCP ModbusClient with pyModbusTCP.
+            The Modbus Client uses a socket connection to facillitate Modbus communications.
+            ModbusClient requires an IP address and port to connect. 
+            
             ModbusClient will not throw errors upon incorrect ip_address!
-            ModbusClient will also allow multiple recconects unlike a socket.
+            
+            ModbusClient auto-reconnects upon socket failure if auto_open is True.
         """
 
         self.scale = ModbusClient(host=ip_address, port=tcp_port, auto_open=True, auto_close=False)
 
+    def decode_scale_weight_registers(register_a, register_b):
```

Comment:
I would still suggest making this a private method by naming it with a leading underscore, i.e. `_decode_scale_weight_registers()`, since it is only used internally to this class.